CHANGELOG.md

```diff
@@ -1,3 +1,47 @@
+# [0.5.0](https://github.com/hyperium/tonic/compare/v0.4.3...v0.5.0) (2021-07-08)
+
+This release includes a new crate `tonic-web` which is versioned at `0.1` and supports accepting `grpc-web`!
+
+### Breaking changes
+
+* `prost` bumped to `0.8`.
+* `BoxBody` was removed in favor of the version provided in `http-body`.
+* The `Connected` trait has been modified to support more generic `ConnectionInfo`.
+* `Streaming` fixed to now return `None` once the stream has observed a trailing `Status`, notifying the user that the stream has ended.
+
+### Bug Fixes
+
+* **build:** fix `with_interceptor` not building on Rust 1.51 ([#669](https://github.com/hyperium/tonic/issues/669)) ([9478fac](https://github.com/hyperium/tonic/commit/9478fac97984cf8291bf89c55eb9a02a06889e03))
```

Comment:
Don't think we need to mention this since this bug was never shipped in the first place. It was introduced in https://github.com/hyperium/tonic/pull/651.